### ISSUE TITLE
demo2 cluster also has the bug #64

### ISSUE THREAD

**Issue by fgeorgatos** [plabrop/bic#66]
This shows a bit better what the original `-x` bug has been about, please fix as per #64 easy bugfix:

```
[fgeorgatos@demo2 ~]$ rpm -qa|grep -i lmod
Lmod-6.5.11-59_cm7.3.noarch
[fgeorgatos@demo2 ~]$ bash -x -c 'echo hello'
+ [[ hBc != hxBc ]]
+ echo 'Start of Lmod init/bash script to define the module command'
Start of Lmod init/bash script to define the module command
+ LMOD_PKG=/usr/share/lmod/lmod
+ LMOD_DIR=/usr/share/lmod/lmod/libexec
+ LMOD_CMD=/usr/share/lmod/lmod/libexec/lmod
+ MODULESHOME=/usr/share/lmod/lmod
+ export LMOD_PKG
+ export LMOD_CMD
+ export LMOD_DIR
+ export MODULESHOME
+ LMOD_VERSION=6.5.11
+ export LMOD_VERSION
+ '[' : '!=' : ']'
+ unalias ml
+ true
++ echo YES
++ /usr/bin/tr '[:upper:]' '[:lower:]'
+ export_module=yes
+ '[' -n '4.2.46(1)-release' -a yes '!=' no ']'
+ export -f module
+ export -f ml
+ unset export_module
+ '[' -n '' ']'
+ '[' 4 -ge 3 ']'
+ '[' -r /usr/share/lmod/lmod/init/lmod_bash_completions ']'
+ '[' -n '' ']'
+ [[ hBc != hxBc ]]
+ echo 'End of Lmod init/bash script to define the module command'
End of Lmod init/bash script to define the module command
+ echo hello
hello
```

**Comment by fgeorgatos** [comment]
the fix should arrive at the following file:
https://github.com/Bright-Computing/bic/blob/master/Lmod/branches/7.0/Lmod-illumina.spec#L129

**Comment by fgeorgatos** [comment]
Fixed via `Lmod/7.1.9` over here:

```
[fgeorgatos@demo2 ~]$ bash -x -c 'echo hello'
+ [[ hBc != hxBc ]]
+ XTRACE_STATE=-x
+ [[ hxBc != hxBc ]]
+ VERBOSE_STATE=+v
+ set +xv
+ unset XTRACE_STATE VERBOSE_STATE
+ echo hello
hello
[fgeorgatos@demo2 ~]$ rpm -qa|grep Lmod
Lmod-7.1.9-63_cm7.3.noarch
[fgeorgatos@demo2 ~]$ module --version

Modules based on Lua: Version 7.1.9  2016-12-17 13:29 -06:00
    by Robert McLay mclay@tacc.utexas.edu
```

**Comment by fgeorgatos** [comment]
bug fixed, therefore closing (we still need the more recent RPMs published though)
